src/Bicep.Cli.E2eTests/eslint.config.mjs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import notice from "eslint-plugin-notice";
+import tsParser from "@typescript-eslint/parser";
+import globals from "globals";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+
+export default [{
+    ignores: ["out/**/*"],
+}, ...compat.extends("eslint:recommended", "plugin:prettier/recommended"), {
+    plugins: {
+        notice,
+    },
+
+    languageOptions: {
```

Comment:
Is `languageOptions` needed for the notice plugin?

Reply:
Discussed in chat